common_rest/src/main/java/uk/gov/dhsc/htbhf/requestcontext/RequestContext.java

```diff
@@ -1,15 +1,45 @@
 package uk.gov.dhsc.htbhf.requestcontext;
 
-import lombok.Data;
-
 /**
- * A container for request-scoped context, such as the id of the current request.
+ * A container for thread context values such as the id of the current request.
  */
-@Data
+@SuppressWarnings("PMD.DataClass")
 public class RequestContext {
```

Comment:
I think we need a `clear()` method here. It's going to be useful at the end of a job, as otherwise information will remain between jobs.

Reply:
And in `RequestIdFilter.doFilterInternal()` - we will need to clear the context between requests at the same time as clearing the MDC context.

Reply:
Clear method added, filter and other common classes updated as well.